src/iop/ashift.c

```diff
@@ -157,7 +157,7 @@ int operation_tags_filter()
   return IOP_TAG_DECORATION | IOP_TAG_CLIPPING;
 }
 
-int default_colorspace(dt_iop_module_t *self,
+dt_iop_colorspace_type_t default_colorspace(dt_iop_module_t *self,
```

Comment:
The parameters should be aligned:

```
dt_iop_colorspace_type_t default_colorspace(dt_iop_module_t *self,
                                            dt_dev_pixelpipe_t *pipe,
                                            dt_dev_pixelpipe_iop_t *piece)
```

Many other instances below.

Reply:
sed doesn't care about formatting ;)
will be done in some further steps

Reply:
aligning parameters done